src/sensor.h

```diff
@@ -197,6 +197,11 @@ namespace librealsense
 
         void create_snapshot(std::shared_ptr<safety_sensor>& snapshot) const override;
         void enable_recording(std::function<void(const safety_sensor&)> recording_function) override {};
+        virtual void set_safety_preset(int index, const rs2_safety_preset& sp) const {}
+        virtual rs2_safety_preset get_safety_preset(int index) const {
+            rs2_safety_preset r;
+            return r;
```

Comment:
Same, virtual = 0, no one will use that..

Reply:
same as above

Reply:
done